### PR TITLE
Feature#23/persistent information

### LINKED ISSUE #23: Persistent information
# REQUIREMENTS
Add blocks of information to the database and that they are in the development and production environments constantly

# Acceptance Criteria
List all required acceptance criteria
- [ ] Run in the seed.rb file.
- [ ] Do not use gems.
- [ ] Same information regardless of the environment.
- [ ] Easy to implement.
- [ ] Easy to replicate.

### PR DESCRIPTION
**Pull request by brayandiazc**
A solution is implemented to have the same information in the development and production database.
Two .csv files were created which can be run from the seed.rb file

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,4 +5,29 @@
...
+CSV.foreach(Rails.root.join("db/seeds/companyBranches.csv"), headers: true) do |row|
+  CompanyBranch.create! do |company_branch|
+    company_branch.id = row[0]
+    company_branch.name = row[1]
+    company_branch.address = row[2]
+  end
+end
+
+puts "Importing employees..."
+CSV.foreach(Rails.root.join("db/seeds/employees.csv"), headers: true) do |row|
```
Comment (fego-dev) [inline db/seeds.rb:22]: You can use faker gem for generate fake data for your application.
https://github.com/faker-ruby/faker :)

Example
https://medium.com/@jvolpe721/using-the-faker-gem-ff3d58ecb2c

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -9,17 +9,16 @@
...
-              <div class="form-floating mb-3">
-                <%= form.text_field :private_number, class:"form-control", id:"floatingInput", placeholder:"" %>
-                <%= form.label :private_number, for:"floatingInput" %>
-              </div>
-              <%= form.submit "Check", class:"w-100 btn btn-lg btn-primary" %>
-              <hr class="my-4">
-              <small class="text-muted">If you do not remember your number it is on the back of your card.</small>
-            <% end %>
-
+          <%= render 'partials/notification' %>
```
Comment (fego-dev) [inline app/views/home/index.html.erb:12]: Nice..

**Review by fego-dev — APPROVED** [review]

**Review by fego-dev — APPROVED** [review]

**Review by Carlos-Montiel — APPROVED** [review]

**Review by fego-dev — APPROVED** [review]

File: db/seeds.rb [hunk 3]
```diff
@@ -5,4 +5,29 @@
...
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
\ No newline at end of file
+AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
+
+require 'csv'
+
+puts "Importing companyBranches..."
+CSV.foreach(Rails.root.join("db/seeds/companyBranches.csv"), headers: true) do |row|
```
Comment (anxhe) [inline db/seeds.rb:13]: What a good idea to avoid spending  time figuring out how to use faker and factory bot. I liked it.

**Review by anxhe — APPROVED** [review]
I liked the csv approach.

**Pull request merged**